src/marten/http/request.cr

```diff
@@ -121,6 +126,10 @@ module Marten
         @request.method.upcase
       end
 
+      protected def method=(method)
+        @request.method = method
+      end
+
```

Comment:
Can we move this method closer to the other `protected` method definitions in this abstraction? 🙏